vendor/tektelic/t0007806-864-profile.yaml

```diff
@@ -0,0 +1,8 @@
+vendorProfileID: 415
```

Comment:
@ssharma-tektelic We have some work in progress (https://github.com/TheThingsNetwork/lorawan-devices/pull/736) that's gonna change how the profile ID works.

I see that you're reusing the same profile ID for multiple different profiles. This will no longer be possible.

Is `415` the actual number you use, or does it happen to be your vendor ID?

Please either use unique numbers for each profile or unset this field.

Reply:
415 is the vendor ID I found in vendor/index.yaml. If I unset it, do I remove the line completely or leave it as "vendorProfileID:"?

Reply:
@ssharma-tektelic remove the line please